package/boot/uboot-mediatek/Makefile

```diff
@@ -34,7 +34,14 @@ define U-Boot/mt7623n_bpir2
   UBOOT_CONFIG:=mt7623n_bpir2
 endef
 
-UBOOT_TARGETS := mt7629 mt7622 mt7623n_bpir2
+define U-Boot/mt7623a_unielec_u7623
```

Comment:
This file could need some sorting. I'd be happy if you just moved the "devices" according to sorting in this patch as well.

Same for the targets line below.

Reply:
Done.